### DISCUSSION TITLE
Way to move all files in folder when mbsync changes an album's name?

### DISCUSSION THREAD

**Discussion by mistwyrm** [beetbox/beets#4682] [Q&A]
I use mbsync and it normally works very well, however I have run into one problem: I like to place my scans of album art into the beets album folders. They're named "front.png", "back.png", "booklet1.png", etc. When mbsync does something that moves the music files into a new folder those images are left behind in the old folder. The mbsync plugin has some code that says it "Moves album art (and any inconsistent items)." but my images seem to be excluded from what it's doing there. Is there something I can do to automate moving my images with the music files?

**Comment by jackwilsdon** [comment]
[beets-extrafiles](https://github.com/Holzhaus/beets-extrafiles) should be able to move these for you.

**Reply by JOJ0** [reply]
@lazybookwyrm have you found the time to give `beets-extrafiles` a try? A word of warning, last time I tried this plugin myself I ran into this problem: https://github.com/Holzhaus/beets-extrafiles/issues/27

There is another plugin that might do what you need: https://github.com/adammillerio/beets-copyartifacts
The original maintainer announced "not maintained" already, but this fork seems to have recent activity and from what the hoster stated in the readme, also some commitment for keeping it maintained. Sounds like worth a try!

HTH
